packages/dnswatch/PKGBUILD

```diff
@@ -0,0 +1,50 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=dnswatch
+pkgver=dnswatch.1.r16.gea62872
+pkgrel=1
+pkgdesc='DNS Traffic Sniffer and Analyzer.'
+arch=('any')
+groups=('blackarch' 'blackarch-sniffer')
+url='https://github.com/HalilDeniz/DNSWatch'
+license=('MIT')
+depends=('python')
+makedepends=('git' 'python-setuptools' 'python-pip')
+source=("$pkgname::git+https://github.com/HalilDeniz/$pkgname.git")
+sha512sums=('SKIP')
+install="$pkgname.install"
```

Comment:
the dependencies are very easy https://github.com/HalilDeniz/DNSWatch/blob/main/requirements.txt, why did you choose a standalone package?

Reply:
Because I gave priority to PKGBUILD Python Standalone template due to the usage of venv. If you think the other PKGBUILD Python template is better, I will change it. Let me know.

Reply:
The rule is like that:

- few dependencies, all dependencies already packaged => [python-PEP517](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-PEP517), [python-setuptools](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-setuptools) or [generic-jointscript](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-jointscript)
- few dependencies, one or two are not packaged yet => same and you package the dependencies with [python-lib-setuptools](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools) or [python-lib-PEP517](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-PEP517)
- very large number of dependencies => [python-standalone](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-standalone)
- dependencies that change very often => [python-standalone](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-standalone)
- requiring outdated version of dependencies, not compatible with the packaged dependencies => [python-standalone](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-standalone)

It's simple: no venv unless necessary

Reply:
gotcha

Reply:
Fixed sir